pkgs/development/tools/build-managers/cmake/default.nix

```diff
@@ -58,6 +61,7 @@ stdenv.mkDerivation rec {
       "--no-system-jsoncpp"
     ]
     ++ optional (!stdenv.isCygwin) "--system-libs"
```

Comment:
What's the reason we use system libs, I would expect cmake to use the libraries from the derivations' buildInputs.